.github/CODEOWNERS

```diff
@@ -104,3 +104,4 @@ src/applications/coronavirus-chatbot @department-of-veterans-affairs/chatbot-adm
 src/applications/vaos @department-of-veterans-affairs/vfs-vaos-fe
 src/applications/dhp-connected-devices @department-of-veterans-affairs/digital-health-platform
 src/applications/virtual-agent @department-of-veterans-affairs/orchid
+src/applications/**/manifest.json @department-of-veterans-affairs/qa-standards
```

Comment:
My only concern with this is teams won't be able to approve changes to their manifests without approval from QA standards

Reply:
Solid point - I'll bring this up with my team during standup tomorrow and circle back to you regarding a solution.

Reply:
Just checked in with my team, we'll make a change to reinstate team ownership once we're done putting the product IDs in.